src/uwtools/drivers/forecast.py

```diff
@@ -6,9 +6,18 @@
 and physics suites.
 '''
 
+import logging
+import os
+import sys
+from datetime import datetime
+import shutil
+
 from .driver import Driver
 
-class SRWFV3Forecast(Driver): # pragma: no cover
+logging.getLogger(__name__)
+
+
+class FV3Forecast(Driver): # pragma: no cover
```

Comment:
Strictly, the forecast.py was setup to include forecasts from all apps, so I put the app name within the class - if we remove this, we'll have to change /scripts/run_forecast.py's autocomplete. I called it 'FV3' as thats how SRW calls it, though MRW just says 'ATM', so some degree of input parsing will be necessary

Reply:
I should have checked to see where the class was used. @christinaholtNOAA requested this change, so hopefully she'll have a chance soon to give guidance on resolution.

Reply:
My hope is that all the Apps will be able to configure the same FV3Forecast object with different parameters. We do want to differentiate between FV3 versus a different forecast model. And we may want to grow the inheritance tree in the future to include a sub-class for a coupled FV3, but I don't think we want to divide this up based on existing Apps. 

@venitahagerty Would you mind changing the instance definition in run_forecast.py to change the naming convention.

Reply:
These different labels define compile-time options for preparing the FV3 forecast executable. Here, we want to make a class that will run all of them. There will be additional settings per UFS App (SRW, MRW, HAFS) that go along with the choice here, but in general ATM is the "simple" one, and then all other compile-time options add on features needed for coupling or additional modeling components for chemistry, waves, ocean, etc.